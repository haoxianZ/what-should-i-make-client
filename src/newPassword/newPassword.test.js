import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NewPassword from './newPassword';

describe(`set new password component`, () => {
  it('renders without crash', () => {
    const match={params:{userId:1}};
    const wrapper = shallow(<NewPassword match={match}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
}) 