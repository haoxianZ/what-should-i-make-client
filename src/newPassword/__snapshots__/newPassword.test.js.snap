// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`set new password component renders without crash 1`] = `
<section
  className="reset"
>
  <form
    onSubmit={[Function]}
  >
    <label
      htmlFor="password"
    >
      New Password:
    </label>
    <input
      autocomplete="new-password"
      id="password"
      name="password"
      required={true}
      type="password"
    />
    <br />
    <br />
    <label
      htmlFor="confirmPassword"
    >
      Confirm Your Password:
    </label>
    <input
      autocomplete="new-password"
      id="confirmPassword"
      name="confirmPassword"
      required={true}
      type="password"
    />
    <button
      className="restBtn"
      type="submit"
    >
      Reset
    </button>
  </form>
</section>
`;
